sample_prof.c

```diff
@@ -90,6 +90,11 @@ static void *sample_prof_handler(void *data) {
 static void sample_prof_start(long interval_usec, size_t num_entries_alloc) {
 	zend_sample_prof_globals *g = SAMPLE_PROF_G;
 
+	if (g->enabled) {
+		zend_throw_exception(NULL, "The profiling is already running", 0);
```

Comment:
Just a phrasing nit:

```suggestion
		zend_throw_exception(NULL, "Profiling is already enabled", 0);
```